src/argus_htmx/management/commands/tailwind_config.py

```diff
@@ -12,38 +15,107 @@
+
+Additional settings that govern the functionality of this command are:
+ * TAILWIND_CONFIG_TARGET: the target location for writing the tailwind.config.js
+ * TAILWIND_CSS_TARGET: override the base tailwind css file location (default: styles.css)
+
 """
-    DEFAULT_TEMPLATE_PATH = "src/argus_htmx/tailwindtheme/tailwind.config.template.js"
-    DEFAULT_TARGET = "src/argus_htmx/tailwindtheme/tailwind.config.js"
+    DEFAULT_CONFIG_TEMPLATE_PATH = "tailwind/tailwind.config.js"
+    DEFAULT_CSS_TEMPLATE_PATH = "tailwind/styles.css"
+
+    # DEFAULT_CONFIG_TARGET = "src/argus_htmx/tailwindtheme/tailwind.config.js"
+    # DEFAULT_CSS_TARGET = "src/argus_htmx/tailwindtheme/styles.css"
+    DEFAULT_CONFIG_TARGET = "tailwind.config.js"
+    DEFAULT_CSS_TARGET = "styles.css"
 
     def handle(self, *args, **options):
-        template_loc = getattr(settings, "TAILWIND_CONFIG_TEMPLATE", self.DEFAULT_TEMPLATE_PATH)
-        target_path = getattr(settings, "TAILWIND_CONFIG_TARGET", self.DEFAULT_TARGET)
+        config_template_name = getattr(
+            settings, "TAILWIND_CONFIG_TEMPLATE", self.DEFAULT_CONFIG_TEMPLATE_PATH
+        )
+        config_target_path = pathlib.Path(
+            getattr(settings, "TAILWIND_CONFIG_TARGET", self.DEFAULT_CONFIG_TARGET)
+        )
+        css_template_name = getattr(
+            settings, "TAILWIND_CSS_TEMPLATE", self.DEFAULT_CSS_TEMPLATE_PATH
+        )
+        css_target_path = pathlib.Path(
+            getattr(settings, "TAILWIND_CSS_TARGET", self.DEFAULT_CSS_TARGET)
+        )
+        self.write_template(
+            config_template_name,
+            config_target_path,
+            context=self.get_context(target_dir=config_target_path.parent),
+            name="tailwind config",
+        )
+        self.write_template(
+            css_template_name,
+            css_target_path,
+            context=self.get_context(target_dir=css_target_path.parent),
+            name="tailwind base css",
+        )
 
-        context = {
-            "themeoverride": getattr(settings, "TAILWIND_THEME_OVERRIDE", argus_htmx_settings.TAILWIND_THEME_OVERRIDE),
+    def get_context(self, target_dir: pathlib.Path):
+        return {
+            "themeoverride": getattr(
+                settings,
+                "TAILWIND_THEME_OVERRIDE",
+                argus_htmx_settings.TAILWIND_THEME_OVERRIDE,
+            ),
             "daisyuithemes": get_themes(),
+            "projectpaths": "\n".join(
+                f"        '{d}/**/*.html'," for d in self.get_template_dirs()
+            ),
+            "cssfiles": (
+                self.make_relative(p, target_dir)
+                for p in sorted(self.get_css_files(), key=lambda p: p.stem)
+            ),
         }
-        template_path = pathlib.Path(template_loc)
-        if not template_path.is_file():
-            self.stdout.write(f"{template_loc} is not a file")
-            return
 
+    def write_template(self, template_name, target_path, context, name):
```

Comment:
You're not writing a template but the result of a rendered template. `write_file` is better.